docs/v2-release-notes.md

```diff
@@ -2,6 +2,16 @@
 
 This page summarizes changes for each major update in Timeplus Enterprise v2 and Timeplus Proton, including new features and important bug fixes.
 
+## Jul 22, 2024
+
+### Timeplus Proton v1.5.14
+  * New feature: Remote user-defined functions (UDFs) are now supported.
+    * Example: `CREATE REMOTE FUNCTION ip_lookup(ip string) RETURNS string URL 'https://abc.lambda-url.us-west-2.on.aws/'`
+
+### Timeplus Enterprise
+  * In the SQL Console, see a query's pipeline after running a query. Note: This is available for single-node on-prem deployments.
+  * New stream mode: mutable streams, where values with the same primary key(s) will be overwritten. More advanced configuration options will be available soon. 
```

Comment:
* New external stream wizard for AutoMQ

Reply:
missing many other features.

* timeplusd
  * feat: [new mutable stream](mutable-stream) for fast UPSERT and high performance point or range query.
  * perf: better asof join performance
  * feat: [parallel key space scan](mutable-stream#key_space_full_scan_threads)
  * feat: force_full_scan for mutable stream
  * feat: user management on cluster
  * fix: [alter stream .. modify settings ..](sql-alter-stream#stream_ttl)
  * feat: support remote UDF on cluster
  * feat: primary key columns in secondary key
  * feat: _tp_message_key to [read/write message keys in Kafka](proton-kafka#messagekey)
  * feat: [Kafka schema registry support for Avro output format](proton-schema-registry#write)
  * feat: collect node free memory usage. You can get it via `select cluster_id, node_id, os_memory_total_mb, os_memory_free_mb, memory_used_mb, disk_total_mb, disk_free_mb, timestamp from system.cluster`
* timeplus_appserver
  * feat: added mutable stream support
  * feat: more metrics for mv stats
  * feat: stream and external stream metrics
  * feat: adapted timeplusd cluster metrics
  * feat: remove ably source
  * feat: improve stats on data lineage
* timeplus_web
  * feat: show more resources stats
  * feat: update license UI
  * feat: show detailed version and build time for components
  * feat: UI for mutable stream CRUD
  * feat: show throughput on data lineage
  * feat: refined GetStarted UI on home page
  * feat: support multinode stats on data lineage
* cli
  * feat: new command for backup/restore data and configuration
  * feat: new command for synchronizing resources to timeplusd
  * feat: when you start Timeplus Enterprise for the first time, auto-create a dashboard to show usage and stats. The template resides in `timeplus/bin/.dashboard`

Reply:
Hi @jovezhong, is autoMQ ready to be shared now?

Reply:
putting in docs release notes is okay. Not decided yet whether to put a English blog for AutoMQ